tools/interop_matrix/client_matrix.py

```diff
@@ -221,6 +221,7 @@ def __init__(self, patch=[], runtimes=[], testcases_file=None):
             ('v1.29.0', ReleaseInfo()),
             ('v1.30.0', ReleaseInfo()),
             ('v1.30.1', ReleaseInfo()),
+            ('v1.30.2', ReleaseInfo()),
```

Comment:
This should replace v1.30.1, and it seems we accidentally didn't replace v1.30.0.

Reply:
Oh, okay. I will remove both 1.30.2 and 1.30.1 and replace 1.30.0 with 1.30.2.